apps/src/Render.cpp

```diff
@@ -122,11 +122,24 @@ static auto GetUVOpts() -> po::options_description
                 "  2 = Orthographic Projection")
         ("uv-reuse", "If input-mesh is specified, attempt to use its existing "
             "UV map instead of generating a new one.")
+        ("uv-align-to-axis", po::value<int>()->default_value(1),
+            "Rotate the UV map so that up (-Y) in the texture image is aligned "
+            "as well as possible to the specified volume axis. Performed "
+            "before uv-rotate and uv-flip.\n"
+            "Volume axis:\n"
+                "  0 = None\n"
+                "  1 = +Z\n"
+                "  2 = -Z\n"
+                "  3 = +Y\n"
+                "  4 = -Y\n"
+                "  5 = +X\n"
+                "  6 = -X")
```

Comment:
I would like the args here to be strings. I can do this change. Just leaving a note.